docs/source/array-stack.rst

```diff
@@ -43,12 +47,38 @@ existing dimension
    >>> import dask.array as da
    >>> import numpy as np
 
-   >>> data = [da.from_array(np.ones((4, 4)), chunks=(2, 2))
-   ...          for i in range(3)]  # small stack of dask arrays
+   >>> arr0 = da.from_array(np.zeros((3, 4)), chunks=(1, 2))
+   >>> arr1 = da.from_array(np.ones((3, 4)), chunks=(1, 2))
+
+   >>> data = [arr0, arr1]
 
    >>> x = da.concatenate(data, axis=0)
    >>> x.shape
-   (12, 4)
+   (6, 4)
 
    >>> da.concatenate(data, axis=1).shape
-   (4, 12)
+   (3, 8)
+
+Block
+-----
+
+We can handle a larger variety of cases with ``da.block`` as it allows
+concatenation to be applied over multiple dimensions at once. This is useful if
+your chunks tile a space. For example, small squares from a larger 2-D plane.
```

Comment:
> For example, small squares from a larger 2-D plane.

Sorry to continue to nitpick, but this is structured as a sentence, but is not actually a sentence.  I would either keep it as a parenthetical, or else expand it to something like "For example, this can be used to tile small squares into a larger 2-D plane"

Reply:
Ouch, sorry, it should have a verb.